views/projects.erb

```diff
@@ -0,0 +1,71 @@
+<html>
+  <head>
+    <title>Food</title>
+    <meta charset = "utf-8">
+  </head>
+
+  <body>
+    <header>
+      <h3>This is where hungry people hang out.</h3>
+      <ul>
+        <li><a href="/">Homepage</a></li>
+        <li><a href ="projects.erb">Projects</a></li>
+        <li><a href="about.erb">About</a></li>
+        <li><a href="blog.erb">Blog</a></li>
+      </ul>
+    </header>
+
+    <div id="kale">
+      <h1>What I made last night</h1>
+      <h3>Kale Salad</h3>
+        <ul>
+          <li>one bunch of kale</li>
+          <li>juice of one lemon</li>
+          <li>parmesan cheese</li>
+          <li>olive oil</li>
+          <li>salt and pepper</li>
+        </ul>
+
+        <ol>
+
+          <li>Wash kale to your standards. I take the extra time to inspect each leaf to make sure I don't end up eating any aphids. But that's a personal preference.</li>
+          <li>With or without aphids, chop kale into bite-sized pieces. Massage your kale if you'd like.</li>
+          <li>Whisk lemon juice, salt and pepper together and then drizzle in olive oil.</li>
+          <li>Pour dressing and shredded parmesan over kale, mix and devour.</li>
+        </ol>
+
+        <img src="http://www.gimmesomeoven.com/wp-content/uploads/2011/09/lemon-parmesan-kale-salad.jpg" alt="Kale Salad" />
+    </div>
+
+    <hr>
+
+    <div id="margarita">
+      <h3>Frozen Honeydew Margarita</h3>
+      <p>Do you find yourself with a surplus of melon in November?<br> Probably not. So file this away for another day.</p>
+        <ul>
+          <li>1 honeydew melon (the green kind, dude)</li>
+          <li>3/4 cup white tequila</li>
+          <li>1/3 cup fresh lime juice</li>
+          <li>sugar, to taste</li>
+        </ul>
+
+        <ol>
+          <li>Slice open your melon and remove the seeds.</li>
+          <li><strong>DO THIS PART 3 HOURS OR A WEEK BEFORE </strong>You are going to freeze the insides of the melon, but you need it to be in bite-sized pieces so you can throw them in the blender. So chop up or scoop out the insides and put them in a freezer bag or something.</li>
+          <li>When you're ready to party, puree in batches frozen honeydew in a blender with all of the ingredients except the sugar.</li>
+          <li>Taste, and blend in sugar if needed.</li>
+        </ol>
+
+        <img src="fresh-honeydew-margaritas.jpg" alt="frozen honeydew margarita" />
+    </div>
+
+
+
+
```

Comment:
Nice job with these! I'd try to reduce some of the extra whitespace that you're using, I'd try to use one line of whitespace for each separation between components